wolfssl/internal.h

```diff
@@ -2538,6 +2538,7 @@ struct WOLFSSL_CTX {
     #ifdef HAVE_ECC
         CallbackEccSign   EccSignCb;    /* User EccSign   Callback handler */
         CallbackEccVerify EccVerifyCb;  /* User EccVerify Callback handler */
+        void*             EccVerifyCtx; /* User EccVerify context */
```

Comment:
You shouldn't need to add `EccVerifyCtx` and `RsaVerifyCtx` here. They are in WOLFSSL.